ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,1045 @@
+    from ocs_ci.ocs.resources import pvc
+
+    storage_class = storage_class or constants.CEPHBLOCKPOOL_SC
+    pvc_obj = pvc.create_pvc(
+        sc_name=storage_class,
+        pvc_name=pvc_name,
+        namespace=namespace,
+        size=size,
+        access_mode=access_mode,
+    )
+    pvc.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND, timeout=300)
+    logger.info(f"Created and bound RBD PVC {pvc_name} in namespace {namespace}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)